### DISCUSSION TITLE
Tables really not working on mobile...

### DISCUSSION THREAD

**Discussion by dgillier** [rappasoft/laravel-livewire-tables#1120] [Feature Requests]
I'm starting to think that tables are not a good idea on mobile with collapsing fields being just trade off...
Same with pagination...

What's about showing  a table on tablets/desktops and a list of cards with inifinite scroll on mobile ?
Searching and filtering will still be available on both mode...

What do you think ?
Denis

**Comment by lrljoe** [comment]
My humble opinion?  I think if the cards PR has sufficient tests, and the same level of functionality as the table (in terms of setting classes etc for specific items), then why not have the option for both, all about more options!
So perhaps configure() options for
- setCardDisplayOnTablet
- setCardDisplayOnMobile
- setTableDisplayOnTablet
- setTableDisplayOnMobile

Then the appropriate implementations for the card to reflect
setTableAttributes/setTheadAttributes/setTbodyAttributes/setThAttributes etc

Then the docs will need updating to refer to the collapseOn() not having a function if you're using the Card Display.

Otherwise, yeah, I don't see why not!

**Reply by lrljoe** [reply]
I've added quite a few comments against the Cards PR #1114 , think that PR has a few things left, would definitely suggest adding this suggestion against the PR tho!
